notebooks/NIRSPEC/BOTS/JWPipeNB-NIRSpec-BOTS.ipynb

```diff
@@ -25,7 +25,7 @@
    "source": [
     "**Authors**: Nikolay Nikolov (AURA Associate Scientist, nnikolov@stsci.edu); Kayli Glidic (kglidic@stsci.edu); NIRSpec branch</br>\n",
     "**Last Updated**: April 18, 2025</br>\n",
-    "**Pipeline Version**: 1.18.0 (Build 11.3, Context jwst_1364.pmap)\n",
+    "**Pipeline Version**: 1.18.1 (Build 11.3, Context jwst_1364.pmap)\n",
```

Comment:
should the context information be here?

Reply:
Builds and pipeline versions are tied to a context by default, so I don't think it hurts to include what the default context is.

Reply:
Hm, I don't think there's any harm in keeping it, but nor is it really necessary here.  A user could certainly change the context below, in which case this note would be incorrect, but a user could equally well open the notebook with a different pipeline software version too.  I'd say we can leave it as a quirk.